pkg/osquery/tables/dev_table_tooling/table.go

```diff
@@ -0,0 +1,74 @@
+package dev_table_tooling
+
+import (
+	"context"
+	"encoding/base64"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+// Encapsulates the binary path(s) of a command allowed to execute
+// along with a strict list of arguments.
+type AllowedCommand struct {
```

Comment:
I think this can be a un-exported
```suggestion
type allowedCommand struct {
```